cv_template/templates/cv_template/cv_template.html

```diff
@@ -91,6 +91,16 @@
     {% csrf_token %}
     {% for form_field in form %}
     {{ form_field|as_crispy_field }}
+    {% if form_field.label == 'Skills' and not user.skills.all %}
+    <p class="text-xs text-gray-600">You must have skills on your profile to create your CV</p>
+    {% elif form_field.label == 'Work experience' and not user.work_experience.all %}
+    <p class="text-xs text-gray-600">You do not have any work experience in your profile</p>
+    {% elif form_field.label == 'Education' and not user.education.all %}
+    <p class="text-xs text-gray-600">You do not have any education in your profile</p>
+    {% elif form_field.label == 'Projects' and not user.projects.all %}
+    <p class="text-xs text-gray-600">You do not have any projects in your profile</p>
```

Comment:
Nitpick (Non-blocking): Not blocking at all, but you don't need the `.all` for these. It could be written like this: 
```
    <p class="text-xs text-gray-600">You must have skills on your profile to create your CV</p>
    {% elif form_field.label == 'Work experience' and not user.work_experience %}
    <p class="text-xs text-gray-600">You do not have any work experience in your profile</p>
    {% elif form_field.label == 'Education' and not user.education %}
    <p class="text-xs text-gray-600">You do not have any education in your profile</p>
    {% elif form_field.label == 'Projects' and not user.projects %}
    <p class="text-xs text-gray-600">You do not have any projects in your profile</p>
    ```

Reply:
The .all is needed here as it checks if there are objects in the field rather than if the field exists